common/test/acceptance/tests/studio/test_studio_library_container.py

```diff
@@ -162,3 +162,54 @@ def test_out_of_date_message(self):
 
         self.assertFalse(library_block.has_validation_message)
         #self.assertIn("4 matching components", library_block.author_content)  # Removed this assert until a summary message is added back to the author view (SOL-192)
+
+    def test_settings_overrides(self):
+        """
+        Scenario: Given I have a library, a course and library content xblock in a course
+        When I go to studio unit page for library content block
+        And when I click the "View" link
+        Then I can see a preview of the blocks drawn from the library.
+
+        When I edit one of the blocks to change a setting such as "display_name",
+        Then I can see the new setting is overriding the library version.
+
+        When I subsequently click to refresh the content with the latest from the library,
+        Then I can see that the overrided version of the setting is preserved.
+
+        When I click to edit the block and reset the setting,
+        then I can see that the setting's field defaults back to the library version.
```

Comment:
As a best practice we do want BDD test descriptions for bok choy tests. The empty rows are not required, but are here for ease of reading (I assume). I think that part of the issue here is that this test is doing a ton of different things, when really it could be broken up into shorter, independent tests.